libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -946,6 +946,8 @@ def dict(self, **kwargs: Any) -> Dict:
     def bind_tools(
         self,
         tools: Sequence[Union[Dict[str, Any], Type[BaseModel], Callable, BaseTool]],
+        *,
+        tool_choice: Union[Literal["any", "auto", "none"], str, None] = None,
```

Comment:
(nit) might make sense to do this in its own PR

Reply:
yea i got a bit carried away